src/pages/collaborations/collaborationManagementForm.jsx

```diff
@@ -55,8 +55,8 @@ export default function CollaborationManagementForm({
           }
           getOptionLabel={option => {
             const name = get(option, 'full_name', null);
-            if (name) return name;
             const email = get(option, 'email', null);
+            if (name) return name + ' (' + email + ')';
```

Comment:
is email guaranteed to be there? if not you could wind up with things like `Peter ()`

Reply:
Good point. If a user doesn't have an email address, though, even for migrated data, we have a bigger problem. But I'll fix.